cime_config/allactive/config_compsets.xml

```diff
@@ -49,6 +49,16 @@
   <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
 </compset>
 
+<compset>
+  <alias>WCYCL1850_chemUCI-Linozv2</alias>
+  <lname>20TRSOI_EAM%CHEMUCI-LINOZV2_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
+</compset>
+
```

Comment:
Do we need coupled compset for Linozv2? Since Linozv3 is the default setting for v3atm

Reply:
You are right, Mingxuan. It is not necessary. But I am leaving it on just like the F-compsets.